conf/ebi_codon.config

```diff
@@ -0,0 +1,54 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Nextflow config file for EMBL-EBI Codon Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Author: Saul Pierotti
+Mail: saul@ebi.ac.uk
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params {
+    config_profile_contact = "Saul Pierotti (@saulpierotti-ebi)"
+    config_profile_description = "The European Bioinformatics Institute HPC cluster (codon) profile"
+    config_profile_url = "https://www.ebi.ac.uk/"
+}
+
+singularity {
+    enabled = true
+    // the default is 20 minutes and fails with large images
+    pullTimeout = "3 hours"
+}
```

Comment:
I'll advise in adding a cacheDir for singularity

Reply:
I see that you recommend in the docs that the location should be
```
"/hps/software/users/<group>/<user_id>/nextflow_software_cache/singularity"
```

I'd reckon it'll be more efficient if shared with all

Reply:
There is not a directory writable by everyone in our cluster...

Reply:
ok, then nevermind